views/article.html

```diff
@@ -67,7 +71,7 @@
 		</script>
 	</head>
 	<body>
-		<amp-analytics config="//SOURCE_HOST{{SOURCE_PORT}}/analytics/config.json">
+		<amp-analytics config="//SOURCE_HOSTNAME{{SOURCE_PORT}}/analytics/config.json">
```

Comment:
We discussed this, and I'm not sure whether a relative path is supported on CDNs (https://www.ampproject.org/docs/reference/extended/amp-analytics.html doesn't make this clear). So I'd prefer to keep it as absolute.